src/v3/src/components/LoopbackProbe/LoopbackProbe.test.tsx

```diff
@@ -240,9 +243,8 @@ describe('LoopbackProbe', () => {
 
     render(<LoopbackProbe {...props} />);
 
-    await waitFor(() => expect(proceedStub).toHaveBeenCalledTimes(1), {
-      timeout: 300, // need to wait longer since we are testing request timeout
-    });
+    // 400ms covers 2 probe requests at 100ms max each plus 1 delayed request at 200ms
+    await waitFor(() => expect(proceedStub).toHaveBeenCalledTimes(1), { timeout: 300 });
```

Comment:
Is the `timeout` here only accounting for the two requests (ignoring the delay)?

Reply:
Ah yeah you're right, the comment doesn't match the actual timeout, _plus_ this should actually have an extra 100ms added. I forgot to include the other `500` failed request for port 6513. I'll update it and the comment to be `500ms`